src/pages/billing/Project.js

```diff
@@ -423,17 +454,35 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
   useEffect(() => { StateHistory.update({ projectUsers }) }, [projectUsers])
   // Update cost data only if report date range changes, or if spend report tab was selected.
   useEffect(() => {
-    const maybeLoadTotalCost = _.flow(
+    const maybeLoadProjectCost = _.flow(
       reportErrorAndRethrow('Unable to retrieve spend report data'),
       Utils.withBusyState(setUpdating)
     )(async () => {
-      if (!updatingTotalCost && totalCost === null && tab === spendReportKey) {
-        setUpdatingTotalCost(true)
+      if (!updatingProjectCost && projectCost === null && tab === spendReportKey) {
+        setUpdatingProjectCost(true)
         const endDate = new Date().toISOString().slice(0, 10)
-        const startDate = new Date((Date.now() - spendReportLengthInDays * 24 * 60 * 60 * 1000)).toISOString().slice(0, 10)
+        const startDate = addDays(-1 * spendReportLengthInDays, new Date()).toISOString().slice(0, 10)
         const spend = await Ajax(signal).Billing.getSpendReport({ billingProjectName: billingProject.projectName, startDate, endDate })
         const costFormatter = new Intl.NumberFormat(navigator.language, { style: 'currency', currency: spend.spendSummary.currency })
-        setTotalCost(costFormatter.format(spend.spendSummary.cost))
+        const categoryDetails = _.find(details => details.aggregationKey === 'Category')(spend.spendDetails)
+        console.assert(categoryDetails !== undefined, 'Spend report details do not include aggregation by Category')
+        const getCategoryCosts = (categorySpendData, asFloat) => {
+          const costDict = {}
+          _.forEach(type => {
+            const costAsString = _.find(['category', type])(categorySpendData)?.cost ?? 0
+            costDict[type] = asFloat ? parseFloat(costAsString) : costFormatter.format(costAsString)
+          }, ['Compute', 'Storage', 'Other'])
+          return costDict
+        }
+        const costDict = getCategoryCosts(categoryDetails.spendData, false)
+        const totalCosts = {
+          spend: costFormatter.format(spend.spendSummary.cost),
```

Comment:
avoid exposing rounding errors, we may wish to obtain the total workspace cost from compute + storage + other. See https://broadinstitute.slack.com/archives/C02L9L96YB1/p1647869664432039?thread_ts=1647818613.734029&cid=C02L9L96YB1.

Reply:
@aherbst-broad I'm leaning towards adding the category values together to get the total cost. What do you think?

Reply:
It's on the server side. I will leave this code as-is, and we can follow up with @marctalbott when he returns. I had [asked him about it](https://broadinstitute.slack.com/archives/C02L9L96YB1/p1647359135280199) while the work was in progress, but then we didn't come to a conclusion.

Here is an example API response from a case that exhibits the issue:
![image](https://user-images.githubusercontent.com/484484/159492316-006d4323-bb0c-4e12-a055-12dc771197c5.png)